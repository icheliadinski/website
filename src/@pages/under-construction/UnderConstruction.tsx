import React from 'react'
import styles from './UnderConstruction.module.scss'

export const UnderConstruction = () => {
  return <div className={styles.card}>Under Construction</div>
}
