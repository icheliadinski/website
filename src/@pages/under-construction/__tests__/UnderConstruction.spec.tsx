import React from 'react'
import { shallow, ShallowWrapper } from 'enzyme'
import { UnderConstruction } from '../UnderConstruction'

describe('[Construct Page]', () => {
  let wrapper: ShallowWrapper

  beforeEach(() => {
    wrapper = shallow(<UnderConstruction />)
  })

  it('renders successfully', () => {
    expect(wrapper).toMatchSnapshot()
  })
})
