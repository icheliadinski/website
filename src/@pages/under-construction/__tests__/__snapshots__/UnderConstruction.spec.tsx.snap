// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[Construct Page] renders successfully 1`] = `
<div
  className="card"
>
  Under Construction
</div>
`;
